vnext/Microsoft.ReactNative.Cxx/JSValueReader-WinRT.h

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <CppWinRTIncludes.h>
+#include <UI.Xaml.Markup.h>
+#include <chrono>
+
+#include "JSValue.h"
+#include "JSValueTreeReader.h"
+#include "StructInfo.h"
+
+#include "winrt/Microsoft.ReactNative.h"
+
+#include <string>
+
+namespace winrt::Microsoft::ReactNative {
+inline BYTE GetAFromArgb(DWORD v) {
+  return ((BYTE)((v & 0xFF000000) >> 24));
+}
+inline BYTE GetRFromArgb(DWORD v) {
+  return ((BYTE)((v & 0x00FF0000) >> 16));
+}
+inline BYTE GetGFromArgb(DWORD v) {
+  return ((BYTE)((v & 0x0000FF00) >> 8));
+}
+inline BYTE GetBFromArgb(DWORD v) {
+  return ((BYTE)((v & 0x000000FF)));
+}
+
+inline ui::Color ColorFromArgb(const JSValue &value) {
```

Comment:
>const [](start = 31, length = 5)

Please use 'right' const as the rest of the code in this project: 'JSValue const&value'. We have adopted the C++/WinRT style in that regard.